src/Symfony/Component/Notifier/Recipient/EmailRecipientTrait.php

```diff
@@ -16,18 +16,9 @@
  *
  * @experimental in 5.1
  */
-class EmailRecipient implements EmailRecipientInterface
+trait EmailRecipientTrait
 {
-    use EmailValidationTrait;
-
-    private $email;
-
-    public function __construct(string $email)
-    {
-        self::ensureEmailIsValid($email);
-
-        $this->email = $email;
-    }
+    protected $email;
```

Comment:
Not sure if this should be protected.

Reply:
I don't think this is needed as one can use the getter.